flux-driver/src/callbacks.rs

```diff
@@ -137,8 +162,13 @@ impl<'a, 'genv, 'tcx> CrateChecker<'a, 'genv, 'tcx> {
         let aliases = specs.aliases;
 
         // Function signatures
-        specs
+        let fn_specs: Vec<(LocalDefId, crate::collector::FnSpec)> = specs
             .fns
+            .into_iter()
+            .filter(|(def_id, _)| !is_ignored(&genv.tcx, &specs.ignores, def_id))
+            .collect();
```

Comment:
I’d follow clippy suggestion here. You can either add this filter to the chain below or early return inside the `try_for_each_exhaust`

Reply:
Where do you see the clippy suggestion? (I had to do this split as the borrow checker was yelling about `tcx` being consumed by the filter...?

Reply:
nvm pushed into the `try_for_each_exhaust`